source/clear-linux/get-started/virtual-machine-install/kvm.rst

```diff
@@ -76,13 +76,13 @@ Download and launch the virtual machine
 
```

Comment:
@eadamsintel. I need your opinion. A user raises a good point: 
The current steps 3-4 conflict, under **Download and launch the virtual machine**.
https://clearlinux.org/documentation/clear-linux/get-started/virtual-machine-install/kvm#kvm

Step 3 asks a user to download the OVMF.fd from image directory (but doesn't explain where or how to use it).
Step 4 asks a user to copy an existing OVMF.fd from directory here: `cp /usr/share/qemu/OVMF.fd`

These are clearly two different methods, and hence, confusing. In my revision, instead, I have the user copy the OVMF.fd from the Downloads dir, keeping with the intent of Step 3. 

Please make a recommendation, or best practice, as to which step, 3 or 4, should be followed. 
I will delete the step that isn't the best practice.  
See also:  https://github.com/clearlinux/clear-linux-documentation/issues/419

Reply:
I'm not sure.  The user said the ovmf files are different so that issue needs to be resolved first. Are they diff because they come from different clear linux releases and it got updated or are they different because a timestamp is part of the binary thus changing every file. 

I would just put a curl command to point to https://cdn.download.clearlinux.org/image/OVMF.fd since that location is not changing. It would cover both cases of using clear linux or using another host.